src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -266,6 +265,46 @@ public Python(String n, String id) {
 
     createPythonInterpreter();
     attachPythonConsole();
+    

```

Comment:
Why did this move here?  seems like this should be in startService...  keep the constructors lightweight.